solidity-frontend/src/main/java/org/sonarsource/solidity/frontend/Utils.java

```diff
@@ -34,4 +35,15 @@ public static SolidityParser returnParserFromParsedFile(CharStream cs) {
     parser.handleComments(tokenStream.getTokens());
     return parser;
   }
+
+  public static void handleStructuredComments(Token token) {
+    // System.out.println(token);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=15&issues=AWQDQR7h3e-qcckjGvIh&open=AWQDQR7h3e-qcckjGvIh)

[ISSUE_KEY]: AWQDQR7h3e-qcckjGvIh